sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2465,12 +2465,20 @@ pi_result piMemBufferCreate(pi_context Context, pi_mem_flags Flags, size_t Size,
     //
   }
 
+  // Choose an alignment is at most 64.
+  auto Alignment = Size;
+  if (Alignment > 64UL)
+    Alignment = 64UL;
+  // When less than 64, it must be a power of 2, else make it 8.
```

Comment:
So this will use 8 for 3 (while could be 4), and 8 for 15 (while should be 16).
Why not just round up to the next pow-2?

Reply:
Alignment calculation has been changed.